libs/SalesforceSDKCore/SalesforceSDKCore/Classes/Util/SFSDKResourceUtils.m

```diff
@@ -54,8 +54,15 @@ + (UIImage *)imageNamed:(NSString *)name
 {
     NSBundle *bundle = [NSBundle bundleForClass:[self class]];
     NSAssert(name != nil, @"name must contain a value.");
+    UIImage *image = [UIImage imageNamed:name inBundle:bundle compatibleWithTraitCollection:nil];
+    if (image) {
+        return image;
+    }
     
-    return [UIImage imageNamed:name inBundle:bundle compatibleWithTraitCollection:nil];
+    // get from main bundle
+    bundle = [NSBundle mainBundle];
```

Comment:
I'm not clear which use case this will support, above and beyond the code that exists.  `SFSDKResourceUtils` is meant to derive its resources from whatever container is most appropriate for *SalesforceSDKCore*.  Which means, in the framework scenario, SalesforceSDKCore.framework, and in the static library scenario, the main bundle.

It's current coding supports those two use cases.  Is there a third case we're not considering?